.github/workflows/ci-test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x]
```

Comment:
Is their any reason/advantage to know that it still builds in Node 12?

Reply:
Not really as far as I know - this line was added 2 years ago where Node 12 had [active support and security support](https://endoflife.date/nodejs). I believe that is just legacy config.